containers.py

```diff
@@ -70,15 +70,14 @@ class LiveStreamContainer(StreamContainer):
     livestream itself, while at the same time caching away previous data in a
     buffer.
     """
-    def __init__(self, vlc_instance, stream_info, buffer_length=200):
+    def __init__(self, vlc_instance, quality, streams, buffer_length=200):
```

Comment:
Maybe we can switch the position on these? Feels more natural with `streams, quality` than the other way around imo :)